Analysis/Core/include/Analysis/TriggerAliases.h

```diff
@@ -0,0 +1,58 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef TriggerAliases_H
+#define TriggerAliases_H
+
+#include <map>
+#include <string>
+#include <vector>
+#include "TNamed.h"
+
+enum triggerAliases {
+  kINT7 = 0,
+  kEMC7,
+  kINT7inMUON,
+  kMuonSingleLowPt7,
+  kMuonUnlikeLowPt7,
+  kMuonLikeLowPt7,
+  kCUP8,
+  kCUP9,
+  kMUP10,
+  kMUP11,
+  kNaliases
+};
+
+class TriggerAliases : public TNamed
```

Comment:
Is there any reason to derive from TNamed?

Reply:
@shahor02 Indeed, it is not necessary. I got inspired by another ccdb-related class by Nicolo
https://github.com/AliceO2Group/AliceO2/blob/dev/Analysis/Core/include/Analysis/RunToTimestamp.h
I changed inheritence to TObject.

Reply:
Actually, it does not need to derive at all (also in https://github.com/AliceO2Group/AliceO2/blob/dev/Analysis/Core/include/Analysis/RunToTimestamp.h), ccdb can store any object with dictionary, extra derivation will just add some unnecessary complexity in its serialization during messaging.

Reply:
I see. I removed TObject inheritance in this commit: https://github.com/ekryshen/AliceO2/commit/3952eeee0213aae41f1bcb001cfa29113d3d96c3
For some reason it wasn't added automatically to this PR. I would suggest to merge this PR and I will make another one.

Reply:
Hi, I will proceed and remove it also from https://github.com/AliceO2Group/AliceO2/blob/dev/Analysis/Core/include/Analysis/RunToTimestamp.h after this PR is merged